string-manipulation-3/src/is-palindromic.js

```diff
@@ -0,0 +1,19 @@
+/* exported isPalindromic */
+function isPalindromic(string) {
+  var leftIndex = 0;
+  var rightIndex = string.length - 1;
+  while (leftIndex < rightIndex) {
+    if (string[leftIndex] === ' ') {
+      leftIndex++;
+    } else if (string[rightIndex] === ' ') {
+      rightIndex--;
```

Comment:
Instead of needing to worry about this, you should just remove the spaces from the string, since they serve no purpose here, and only create additional work for you.